package.json

```diff
@@ -166,7 +166,7 @@
     "karma-junit-reporter": "^1.2.0",
     "ng-packagr": "^9.0.0",
     "npm-package-versions": "^1.0.1",
-    "prettier": "1.19.1",
+    "prettier": "2.0.2",
```

Comment:
I'd prefer adding the major with caret, and leave the patch (at least) up to the lock file. 

`"prettier": "^2.0.0",`

Reply:
👍